### PR TITLE
adding netlify cms to admin path

### PR DESCRIPTION
**Pull request by sebbsan**
What does this PR do?

- It's adding Netlify CMS to the /admin path
- If you have a Git account for this Repo, it will allow to log you in
- you can edit content - right now the Privacy Page - which is stored in Markdown Files, and FAQ, which is stored in JSON file

### CODE REVIEW

**Comment by manzke** [comment]
@sebbsan what's the status?

**Comment by ajantsch** [comment]
What I can't see is excluding `public/admin` path from a prod deployment.

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ jobs:
         run: |
           yarn install
           yarn build
+          rm -rf /build/admin
```
Comment (ajantsch) [inline .github/workflows/deploy.yml:21]: I added this to not deploy the admin to production

File: src/components/common/MuiMarkdown.tsx [hunk 2]
```diff
@@ -56,7 +65,7 @@ class MuiMarkdown extends React.PureComponent {
           },
         }}
       >
-        {this.props.children}
+        {this.sanitisedMarkdown}
```
Comment (ajantsch) [inline src/components/common/MuiMarkdown.tsx]: This doesn't work as it doesn't change the used markdown when language is change, you have to make it part of the state.

Comment (ajantsch) [reply src/components/common/MuiMarkdown.tsx]: Fixed with https://github.com/gesund-zusammen/gesund-zusammen.github.io/pull/93/commits/19cbf62a6b8faec2feeab2f33e1fabe386a74477

File: src/components/common/MuiMarkdown.tsx [hunk 3]
```diff
@@ -2,8 +2,17 @@ import React from "react";
...
 import Markdown from "markdown-to-jsx";
 
-class MuiMarkdown extends React.PureComponent {
+interface IMuiMarkdownProps {
+  markdown: string;
+}
+
+class MuiMarkdown extends React.PureComponent<IMuiMarkdownProps, {}> {
   render = () => {
+    const frontMatterRegExp = /(\-\-\-(.|\n)*\-\-\-)/;
```
Comment (manzke) [inline src/components/common/MuiMarkdown.tsx:11]: Shouldn't we strip this out during build time? :)

Comment (sebbsan) [reply src/components/common/MuiMarkdown.tsx:11]: I agree.. let me dive into WebPack once more..

Comment (sebbsan) [reply src/components/common/MuiMarkdown.tsx]: one done in the latest commit 👍

**Pull request merged**